index.html

```diff
@@ -392,13 +392,13 @@ <h3>Information for User Agents</h3>
 	      <li>Set the <abbr title="Document Object Model">DOM</abbr> focus:
 		<ol>
 		  <li>If the <a class="termref">element</a> can take <abbr title="Document Object Model">DOM</abbr> focus, the <a class="termref">user agent</a> MUST set the <abbr title="Document Object Model">DOM</abbr> focus to it.</li>
-		  <li>Otherwise, if the current element has an ID and an ancestor with the <pref>aria-activedescendant</pref> attribute present, and that ancestor is focusable, the user agent MUST set <abbr title="Document Object Model">DOM</abbr> focus to that ancestor.
+		  <li>Otherwise, if the current element has an ID and is <a>owned</a> by a container element with the <pref>aria-activedescendant</pref> attribute present, and the containter element is focusable, the user agent MUST set <abbr title="Document Object Model">DOM</abbr> focus to the container element.
```

Comment:
```suggestion
		  <li>Otherwise, if the current element has an ID and is <a>owned</a> by either a container element that has the <pref>aria-activedescendant</pref> property or by a container element that is controlled by an element that has the <pref>aria-activedescendant</pref> property, and the element that has <pref>aria-activedescendant</pref> is focusable, the user agent MUST set <abbr title="Document Object Model">DOM</abbr> focus to the element that has <pref>aria-activedescendant</pref>.
```